wicketstuff-restannotations-parent/restannotations/src/main/java/org/wicketstuff/rest/resource/AbstractRestResource.java

```diff
@@ -496,7 +493,9 @@ private void throwAmbiguousMethodsException(ScoreMethodAndExtractPathVars... met
 				methodsNames += ", ";
 
 			MethodMappingInfo urlMappingInfo = method.getMethodInfo();
-			methodsNames += urlMappingInfo.getMethod().getName();
+			methodsNames += urlMappingInfo.getMethod().getReturnType().getSimpleName() + " " +
```

Comment:
Maybe it worth to use `StringBuilder` in this method?

Reply:
changed